src/main/java/javax/resource/spi/LocalTransactionException.java

```diff
@@ -19,7 +19,7 @@
 /** 
  * A <code>LocalTransactionException</code> represents various 
  * error conditions related to the local transaction management contract. 
- * The Java Transaction API specification specifies the 
+ * The Jakarta&trade; TransactionsI specification specifies the 
```

Comment:
Minor typo with extra 'I' at end of `Transactions`

Reply:
Fixed